pkg/util/timeutil/time_zone_util.go

```diff
@@ -68,3 +74,49 @@ func ParseFixedOffsetTimeZone(location string) (offset int, origRepr string, suc
 	}
 	return offset, strings.TrimSuffix(strings.TrimPrefix(origRepr, "("), ")"), true
 }
+
+// TimeZoneOffsetStringConversion converts a time string to offset seconds.
+// Supported time zone strings: GMT/UTC±[00:00:00 - 169:59:00].
+// Seconds/minutes omittable and is case insensitive.
+func TimeZoneOffsetStringConversion(s string) (offset int64, ok bool) {
+	timeZoneString := strings.ReplaceAll(s, " ", "")
+	if !(timezoneOffsetRegex.MatchString(timeZoneString)) {
```

Comment:
Do we need this check if we check Len of submatch below?

Reply:
I guess not. Changing now